app/controllers/users_controller.rb

```diff
@@ -0,0 +1,15 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+    @posts = @user.posts
+  end
```

Comment:
- Good job done so far, but according to the project requirements, you are required to use the `#recent_post` method in the User model to render the user's 3 most recent posts. Please do this instead of hiding the overflow 😄